README.md

```diff
@@ -104,7 +104,7 @@ docker run -e SURVEY_RUNNER_SCHEMA_URL=http://docker.for.mac.host.internal:5000
 To submit data you will also need to run an additional upstream tool (eg, https://github.com/ONSDigital/eq-docker-dynamodb) to launch a dynamoDB container.
```

Comment:
```suggestion
The data storage backend is configurable with the `EQ_STORAGE_BACKEND` environment variable. You will need to run an emulator if you are setting this to DynamoDB or Google Datastore:

DynamoDB: https://github.com/ONSDigital/eq-docker-dynamodb
Google Datastore: https://hub.docker.com/r/knarz/datastore-emulator/
```

Reply:
There is a list of env variables in the readme which appears to be out of date (even prior to this PR). I'll raise a separate ticket to cover that.. and to introduce the new env vars delivered by this PR